scolapasta-path/src/lib.rs

```diff
@@ -34,7 +34,7 @@
 mod paths;
```

Comment:
Could you bump scolapasta-paty to 0.5.0?

You'll have to change its `Cargo.toml`, `README.md`, any direct deps in the various `Cargo.toml`s in the Artichoke workspace, and then run `bundle exec rake build:all` to bump the fuzz and spec-runner lockfiles.

Please do all of these steps in a single, separate commit.